pyincore/analyses/seasidecge/seasidecge.py

```diff
@@ -621,46 +621,46 @@ def _(x):
         # PARAMETERS AND ELASTICITIES
```

Comment:
Connection successful to IN-CORE services. pyIncore version detected: 1.6.0
/Users/cwang138/Documents/INCORE-2.0/pyincore/pyincore/analyses/seasidecge/seasidecge.py:779: FutureWarning: In a future version, `df.iloc[:, i] = newvals` will attempt to set the values inplace instead of always setting a new array. To retain the old behavior, use either `df[df.columns[i]] = newvals` or, if columns are non-unique, `df.isetitem(i, newvals)`
  IGT0.loc[G, GX] = SAM.loc[G, GX]
/Users/cwang138/Documents/INCORE-2.0/pyincore/pyincore/analyses/seasidecge/seasidecge.py:852: FutureWarning: In a future version, `df.iloc[:, i] = newvals` will attempt to set the values inplace instead of always setting a new array. To retain the old behavior, use either `df[df.columns[i]] = newvals` or, if columns are non-unique, `df.isetitem(i, newvals)`
  B.loc[I, IG] = BB.loc[I, IG].fillna(0.0)

I got warnings here

Reply:
That is weird, I am not getting the warnings on running the tests
![image](https://user-images.githubusercontent.com/16596733/192896578-5d75e0be-98f7-4432-bd6b-32e723a63900.png)